### DISCUSSION TITLE
动画卡住

### DISCUSSION THREAD

**Discussion by maleah714** [Tencent/libpag#1987] [Q&A]
同一个pagview由某一个循环动画转变为单次播放的动画时，动画会卡在单次播放的动画的第一帧，不会播放单次动画。
com.tencent.tav:libpag:4.2.41版本不存在该问题，com.tencent.tav:libpag:4.3.33版本出现这个问题。
复现demo及视频在下面的链接中：
链接：https://pan.baidu.com/s/18vCLwCQu3LyBSP5k9SnVSg 
提取码：48p2
说明：复现demo中是在A Simple PAG Animation页面先循环播放replacement.pag动画，然后在3s后播放test2.pag动画一次，结果动画停留在test2.pag动画的第一帧，没有播放test2.pag动画。改为循环播放test2.pag动画没有问题，只有单次播放pagView.setRepeatCount(0)这样会卡住。

**Comment by CodeJhF** [comment]
这个问题当时定位在 4.2.41 / 4.3.33 附近，之后 libpag 迭代了多个版本，先关闭本条。
麻烦在最新版本（4.4 / 4.5）上再复现一次：如果仍能复现，请回复版本号 + 最小 demo，我们会重新排期处理；如果已经正常，本条就不用再跟了。（stale-close）
